app/lib/formatter.rb

```diff
@@ -19,6 +19,10 @@ def format(status, **options)
 
     raw_content = status.text
 
+    if status.poll && options[:inline_poll_options]
```

Comment:
The other way around is better

Reply:
You mean, checking the option then the poll? Or have it on by default? Changed it for the former.